src/lib/browser/BrowserPage.ts

```
@@ -668,6 +670,224 @@ export class BrowserPage {
+    // Prefer CDP (what's actually loaded in the tab) to avoid network/referrer variance
+    try {
+      const bytes = await this._fetchPdfBytesViaDebugger(url)
+      if (bytes && bytes.byteLength > 0) {
+        const magic = new TextDecoder('latin1').decode(bytes.slice(0, 5))
+        if (!magic.startsWith('%PDF-')) throw new Error('NON_PDF_CONTENT: first bytes are not %PDF')
+        return bytes
+      }
+      throw new Error('DEBUGGER_FAILED: No content')
+    } catch (e) {
+      // Fallback to credentialed fetch
+      let prior = e instanceof Error ? e.message : String(e)
```

ESLint

[error] 'prior' is never reassigned. Use 'const' instead.

(prefer-const)